client/src/components/Card/Card.css

```diff
@@ -12,57 +12,65 @@
   font-family: var(--default-font);
   min-height: 9.5rem;
 }
+
 .Card--purple {
   background-color: var(--purple);
 }
+
 .Card--pink {
   background-color: var(--pink);
 }
+
 .Card--red {
   background-color: var(--red);
 }
+
 .Card--pink {
   background-color: var(--pink);
 }
+
 .Card--yellow {
   background-color: var(--yellow);
 }
+
 .Card--large {
   width: 32vw;
   max-width: 500px;
   min-width: 200px;
   max-height: 275px;
```

Comment:
This is a little odd - do we need to set max widths, min-widths, a width, and a max height? We definitely want this to have a minimum size if it's a large card